deploy/helm/charts/templates/zfs-controller.yaml

```diff
@@ -118,7 +118,7 @@ spec:
                 fieldRef:
                   fieldPath: metadata.namespace
             - name: OPENEBS_IO_INSTALLER_TYPE
-              value: "zfs-localpv-helm"
+              value: "{{ .Values.analytics.installerType }}"
```

Comment:
ensure it's not a breaking change when using old values yaml

```suggestion
              value: "{{ default .Values.analytics.installerType "zfs-localpv-helm" }}"
```

Reply:
Doesn't seem to work.